src/components/cluster/Cluster.tsx

```diff
@@ -666,6 +666,7 @@ const Cluster = ({
     return (
         <>
             <article
+                data-testid={`${cluster_name}-cluster-container`}
                 className={`cluster-list ${
                     clusterId ? 'cluster-list--update' : 'cluster-list--create collapsed-list collapsed-list--create'
```

Comment:
What if cluster_name is not available? Also check if we can use clusterId instead of name as the name can be changed by the user

Reply:
But still it can be undefined.

Reply:
default cluster should be there always.